examples/js/loaders/FBXLoader.js

```diff
@@ -4329,7 +4329,7 @@
 
 					if ( window.Zlib === undefined ) {
 
-						throw new Error( 'FBXLoader: Import inflate.min.js from https://github.com/imaya/zlib.js' );
+					  throw new Error('FBXLoader: External library Inflate.min.js required, obtain or import from https://github.com/imaya/zlib.js');
```

Comment:
Thanks for contributing! When you make a PR, please respect our code style: https://github.com/mrdoob/three.js/wiki/Mr.doob%27s-Code-Style%E2%84%A2.

In your case:

- Indentation should be tabs (you use just spaces)
- There should be a single space after the opening bracket and  before the closing bracket.